src/django_components/component.py

```diff
@@ -131,6 +132,47 @@ def all_components() -> List[Type["Component"]]:
     return components
 
 
+# NOTE: Initially, we fetched components by their registered name, but that didn't work
```

Comment:
This comment, as well as the `comp_cls_id_mapping` dictionary, were previously in `dependencies.py`.